### DISCUSSION TITLE
Provide docker image tag latest to link to the latest image release

### DISCUSSION THREAD

**Discussion by klimeto** [INSPIRE-MIF/helpdesk-validator#1006] [Ideas]
Dear helpdesk,

Would it be possible to configure docker registry returning the last version of `inpsire-validator` image while pulling with the following command:

```bash
docker pull docker.pkg.github.com/inspire-mif/helpdesk-validator/inspire-validator
Using default tag: latest
Error response from daemon: manifest for docker.pkg.github.com/inspire-mif/helpdesk-validator/inspire-validator:latest not found: manifest unknown: Docker image reference inspire-validator:latest not found under repo "inspire-mif"
```

My goal is to be able pulling always the newest version without a need to manually check the version tag, which is currently `2021.1`.

Thank you!

**Comment by fabiovin** [comment]
Dear @klimeto,

sorry for the late reply.

We’re analyzing your proposal, and we’ll let you know as soon as possible.

Thanks,
Fabio

**Comment by bcosyn** [comment]
We are working on a way to deploy a local version of the validator on MS Azure. It would be a great idea so we can move to automatic deploy and offer always the latest validator.

**Comment by jenriquesoriano** [comment]
Dear @klimeto , @bcosyn , @joachimnielandt,

thank you for this is a very interesting proposal for improving the INSPIRE Reference Validator, as it would enable continuous and automated deployment of the Docker image in your premises.
We are moving this issue to a discussion to gather further feedback from the community.

Thank you very much for the contribution.
